org.opent2t.sample.binaryswitch.superpopular/com.smartthings.binaryswitch/js/tests/test.js

```diff
@@ -24,10 +24,10 @@ function getBinarySwitch(devices) {
 // setup the translator before all the tests run
 test.before(async () => {
     var hubTranslator = await OpenT2T.createTranslatorAsync(hubPath, 'thingTranslator', config);
-    var hubInfo = await OpenT2T.invokeMethodAsync(hubTranslator, 'org.opent2t.sample.hub.superpopular', 'get', [false]);
```

Comment:
Should I change the test files occurring to Chuck's PR right now?

---
In reply to: [90384586](https://github.com/openT2T/translators/pull/134#discussion_r90384586) [](ancestors = 90384586)